### ISSUE TITLE
[FR] (feature summary) UBL mesh edit LCD interface

### ISSUE THREAD

**Issue by apulver** [MarlinFirmware/Marlin#25260]
### Describe the feature you want

When editing a mesh using "Mesh Edit" from the UBL menu, it would be convenient if, after editing a mesh value, the user were returned to the graphical "grid" where points can be viewed and selected, rather than automatically moved along to the next adjacent point to edit.

**Comment by borland1** [comment]
Are you sure about that?  This 3-year old Chris Riley YouTube.com video shows him using UBL edit mesh with LCD menu..  each point selectable individually. 

[Marlin Unified Bed Leveling - How To - Chris's Basement](https://www.youtube.com/watch?v=ONpKxkil16Q&t=376s)

![image](https://user-images.githubusercontent.com/3272058/213929643-2f6adb98-6338-4e82-88fd-3c74545a3f78.png)

**Comment by apulver** [comment]
> Are you sure about that? This 3-year old Chris Riley YouTube.com video shows him using UBL edit mesh with LCD menu.. each point selectable individually.

Yes, you can select each point individually, but then you are taken to every adjacent point. It would be convenient to have the option of editing just one single point before the user is returned to the grid.
